ai_worker/src/parsers/text.py

```
@@ -0,0 +1,108 @@
+            str: 추출된 텍스트
+
+        Raises:
+            ImportError: PyPDF2가 설치되지 않았을 때
+        """
+        try:
+            from PyPDF2 import PdfReader
+        except ImportError:
+            raise ImportError(
+                "PyPDF2 is required for PDF parsing. "
+                "Install it with: pip install PyPDF2"
+            )
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)